### ISSUE TITLE
NSComparisonResult Unexpected Identifier

### ISSUE THREAD

**Issue by bassrock** [drkameleon/iswift#126]
I have a category declaration like so on the top of one of my classes:

```obj-c
// A category on NSNetService that's used to sort NSNetService objects by their name.
@interface NSNetService (BrowserViewControllerAdditions)
- (NSComparisonResult) localizedCaseInsensitiveCompareByName:(NSNetService*)aService;
@end

@implementation NSNetService (BrowserViewControllerAdditions)
- (NSComparisonResult) localizedCaseInsensitiveCompareByName:(NSNetService*)aService {
	return [[self name] localizedCaseInsensitiveCompare:[aService name]];
}
@end
```

The converter is hanging up on NSComparisonResult, Not sure if it is because it is a category or if the converter does not know how to handle it.

**Comment by drkameleon** [comment]
Well, I have to say it's (yet another) 'unknown type' error... :S

If you change it to `id` (or anything else for that matter), it will work.

In any case, it's in my to-do list and it'll be fixed by the upcoming version -- meaning: **really** soon. ;-)

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 2.1

**Example :**

```objective-c
// A category on NSNetService that's used to sort NSNetService objects by their name.
@interface NSNetService (BrowserViewControllerAdditions)
- (NSComparisonResult) localizedCaseInsensitiveCompareByName:(NSNetService*)aService;
@end

@implementation NSNetService (BrowserViewControllerAdditions)
- (NSComparisonResult) localizedCaseInsensitiveCompareByName:(NSNetService*)aService {
    return [[self name] localizedCaseInsensitiveCompare:[aService name]];
}
@end
```

**Output:**

```swift
extension NSNetService {

	func localizedCaseInsensitiveCompareByName(aService: NSNetService) -> NSComparisonResult {
		return self.name().localizedCaseInsensitiveCompare(aService.name())
	}

}

```
